applets/wncklet/org.mate.panel.applet.window-list.gschema.xml.in

```diff
@@ -10,6 +10,16 @@
       <summary>Show windows from all workspaces</summary>
       <description>If true, the window list will show windows from all workspaces. Otherwise it will only display windows from the current workspace.</description>
     </key>
+    <key name="show-window-thumbnails" type="b">
+      <default>true</default>
+      <summary>Display window thumbnails on hover</summary>
+      <description>If true, then when hovering over a taskbar item, a thumbnail of the window will appear. It will go away as soon as the mouse leaves the item.</description>
```

Comment:
it needs to add "requires libwnck 3.32.0" in description?

Reply:
Why? I never saw a version in dconf key description.

Reply:
the feature doesn't work in lower versions, the user must know it in my opinion

Reply:
This is the wrong place to do this.
Feel free to write release notes for 1.24, this is the proper place to inform users about a change.